ansible_ai_connect/ai/api/model_pipelines/wca/pipelines_saas.py

```diff
@@ -427,6 +427,11 @@ class WCASaaSRoleExplanationPipeline(
     def __init__(self, config: WCASaaSConfiguration):
         super().__init__(config=config)
 
+    def invoke(self, params: RoleExplanationParameters) -> RoleExplanationResponse:
+        if not settings.ANSIBLE_AI_ENABLE_ROLE_GEN_ENDPOINT:
+            raise NotImplementedError
```

Comment:
Sorry, I missed this the first time.

This should now be `FeatureUnavailable` (if you've sync'ed with `main` you'll see the other pipelines/functions do that now). Remember my PR?